### DISCUSSION TITLE
Add Anime movie category like done for Series

### DISCUSSION THREAD

**Discussion by DevYukine** [sct/overseerr#2218] [Feature Requests]
Hello 👋 I really enjoy the option to add certain tags only for anime which is there for Series but i would love to have the same thing for movies as exactly the same issue it fixed for me on Series (Different Groups / Rules by tag and the issues which come with that for finding/sorting releases in Sonarr/Radarr etc) now start to appear on requested anime movies for me as well and i was hoping this could be easily solved by the same system 😄.

Hope to get some feedback is this is possible/planned at all.

**Comment by TheCatLady** [comment]
You can subscribe to #1881 for updates. (Please do not comment on the issue and spam everyone unless you have something constructive to add.)

**Reply by DevYukine** [reply]
This isn't exactly what i want tho, my anime movies and movies are on the same Radarr Server and only change by a certain tag i added and i would love to keep it this way as it already works this way for Sonarr, thats why i suggested making the same change like for Series

EDIT: i just read that it will be solved differently than described in the title so thats better 😄 i will watch that issue then, thanks

**Reply by TheCatLady** [reply]
If you read the `Desired Behavior` section, it is exactly how you described.  However, we don't plan to implement this the same way for Radarr.  Support will come along with a more flexible system of applying paths/profiles/tags to different media.
